ush/rocoto/setup_workflow.py

```diff
@@ -1030,6 +1058,25 @@ def get_hyb_tasks(dict_configs, cycledef='enkf'):
 
         dict_tasks[f'{cdump}eupd'] = task
 
+        # # eupdfsoi
+        do_efsoi = base.get('DO_EFSOI', 'NO').upper()
+
+        if do_efsoi in ['Y', 'YES']:
+
+            deps = []
+            if lobsdiag_forenkf in ['.F.', '.FALSE.']:
+                dep_dict = {'type': 'metatask', 'name': '%seomn' % cdump}
```

Comment:
Please use modern python string formatting (preferably f-strings) and not the old % notation. Here and throughout the rest of the script.

Reply:
done